FRv2.1 GA Stable Release

sandeepparab opened the pull request:
- FRv2.1 GA Stable Release

Commit:
Standardized Errors, sample list.

Commit:
removing from master from now as this is just a proposal

Commit:
Merge branch 'master' of https://github.com/azure/azure-rest-api-specs

Commit:
Merge branch 'master' of https://github.com/azure/azure-rest-api-specs

Commit:
FRv2.1 GA Initial commit.

jhendrixMSFT commented:
@sandeepparab has this new API version been reviewed and approved by the API review board?

sandeepparab commented:
Hi @jhendrixMSFT, since v2.1 GA has no swagger changes since v2.1-preview.3 it is assumed that we are good to go with this. Please let me know if I got this wrong. 

Thanks much!
-Sandeep
